### PR TITLE
JAMES-2184 MPT test to show delegated mailboxes are not IMAP listed

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by mbaechler** [comment]
test this please

File: mpt/core/src/main/java/org/apache/james/mpt/script/ImapScriptedTestProtocol.java [hunk 1]
```diff
@@ -37,7 +38,25 @@ public void prepare(ImapHostSystem system) throws Exception {
...
+        final String userName;
+        final MailboxACL.Rfc4314Rights rights;
+
+        public CreateRights(MailboxPath mailboxPath, String userName, MailboxACL.Rfc4314Rights rights) {
+            this.mailboxPath = mailboxPath;
+            this.userName = userName;
+            this.rights = rights;
+        }
+
+        public void prepare(ImapHostSystem system) throws Exception {
```
Comment (strokyl) [inline mpt/core/src/main/java/org/apache/james/mpt/script/ImapScriptedTestProtocol.java]: Please add ``@Override``

File: mpt/impl/imap-mailbox/core/src/main/java/org/apache/james/mpt/imapmailbox/suite/ListingWithSharing.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import org.apache.james.mailbox.model.MailboxACL;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mpt.api.ImapHostSystem;
+import org.apache.james.mpt.imapmailbox.ImapTestConstants;
+import org.apache.james.mpt.imapmailbox.suite.base.BasicImapCommands;
+import org.apache.james.mpt.script.ImapScriptedTestProtocol;
+import org.junit.Before;
+import org.junit.Test;
+
+public abstract class ListingWithSharing implements ImapTestConstants {
```
Comment (strokyl) [inline mpt/impl/imap-mailbox/core/src/main/java/org/apache/james/mpt/imapmailbox/suite/ListingWithSharing.java]: ListingWithSharingTest ?

**Comment by rouazana** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**